packages/server-admin-ui/src/views/appstore/appStore.scss

```
@@ -154,3 +154,386 @@
+  border-left: 1px solid #e4e7ea;
+  margin-left: 12px;
+  padding-left: 10px;
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+  gap: 6px;
+  font-family:
+    SFMono-Regular,
+    Menlo,
+    Consolas,
+    Liberation Mono,
```

Stylelint (17.9.0)

[error] Expected quotes around "Liberation Mono" (font-family-name-quotes)

(font-family-name-quotes)